src/index.ts

```diff
@@ -1550,7 +1574,8 @@ export const formats: IFormat[] = [
   hexChecksumChain('GO_LEGACY', 6060),
   bech32mChain('XCH', 8444, 'xch', 90),
   getConfig('NULS', 8964, nulsAddressEncoder, nulsAddressDecoder),
-  bech32Chain('AVAX', 9000, 'avax'),
+  getConfig('AVAX', 9000, makeBech32Encoder('avax'), makeAvaxDecoder('avax')),
```

Comment:
Can you check only X or P are allowed?

Reply:
Based on the documentation (https://support.avax.network/en/articles/4596397-what-is-an-address), X or P are allowed and are technically separate chains using the same addressing format (besides the prefix).

There seems to be a new feature, the `chainId` can be used instead of an alias:
> Same address, using the chainID instead of the chain alias:
11111111111111111111111111111111LpoYY-avax1kj06lhgx84h39snsljcey3tpc046ze68mek3g5

Is this something you would like to see supported too?